### PR TITLE
Remove isScalar (use numpy instead)

### PR DESCRIPTION
**Pull request by rowanc1**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -43,45 +43,72 @@ def test_ndgrid_2D(self):
...
         nN = np.array([3, 3, 3])
-        self.assertTrue(np.all(indexCube('A', nN) == np.array([0, 1, 3, 4, 9, 10, 12, 13])))
-        self.assertTrue(np.all(indexCube('B', nN) == np.array([3, 4, 6, 7, 12, 13, 15, 16])))
-        self.assertTrue(np.all(indexCube('C', nN) == np.array([4, 5, 7, 8, 13, 14, 16, 17])))
-        self.assertTrue(np.all(indexCube('D', nN) == np.array([1, 2, 4, 5, 10, 11, 13, 14])))
-        self.assertTrue(np.all(indexCube('E', nN) == np.array([9, 10, 12, 13, 18, 19, 21, 22])))
-        self.assertTrue(np.all(indexCube('F', nN) == np.array([12, 13, 15, 16, 21, 22, 24, 25])))
-        self.assertTrue(np.all(indexCube('G', nN) == np.array([13, 14, 16, 17, 22, 23, 25, 26])))
-        self.assertTrue(np.all(indexCube('H', nN) == np.array([10, 11, 13, 14, 19, 20, 22, 23])))
+        assert np.allclose(
```
Comment (rowanc1) [inline tests/test_utils.py:88]: This file is mostly pep8 updates.

File: tests/test_utils.py [hunk 2]
```diff
@@ -135,6 +154,35 @@ def test_asArray_N_x_Dim(self):
         self.assertTrue(np.all(true == listArray))
         self.assertTrue(true.shape == listArray.shape)
 
+    def test_mesh_tensor(self):
```
Comment (rowanc1) [inline tests/test_utils.py:157]: except for some additions in testing `mesh_tensor`

**Review by rowanc1 — COMMENTED** [review]
Remove isScalar

**Review by lheagy — APPROVED** [review]
good call, thanks @rowanc1!

**Pull request merged**